client/packages/system/src/Patient/PatientView/AddButton.tsx

```diff
@@ -60,12 +61,18 @@ export const AddButton = () => {
       <ProgramSearchModal
         disabledPrograms={data?.nodes?.map(program => program.type)}
         open={current === PatientModal.ProgramSearch}
-        onClose={() => setCurrent(undefined)}
+        onClose={() => reset()}
```

Comment:
using `() => reset()` creates a new method on every render. It is more efficient to do this:

```suggestion
        onClose={reset}
```